storage/storage_image.go

```diff
@@ -537,8 +538,9 @@ func (s *storageImageSource) GetTargetManifest(digest ddigest.Digest) (manifestB
 	return nil, "", ErrNoManifestLists
 }
 
-func (s *storageImageSource) GetSignatures() (signatures [][]byte, err error) {
+func (s *storageImageSource) GetSignatures(ctx context.Context) (signatures [][]byte, err error) {
 	var offset int
+	// FIXME: pass the context.Context
```

Comment:
contaiers/storage is generally dealing with local files, ignoring the `ctx` here should be fine.